lib/query_helper/sql_sort.rb

```diff
@@ -31,7 +32,15 @@ def attributes_sql_expression(sort_attribute)
         direction = sort.split(":")[1]
         modifier = sort.split(":")[2]
         begin
-          sql_expression = @column_maps.find{ |m| m.alias_name.casecmp?(sort_alias) }.sql_expression
+          if @column_sort_order.present?
+            sql_expression = '(CASE'
+            @column_sort_order.each_with_index do |type, index|
```

Comment:
Should we sanitize the query parameters to avoid injection attacks?
https://api.rubyonrails.org/classes/ActiveRecord/Sanitization/ClassMethods.html

Also, there are no tests on this commit. Please, add tests.